Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:31:20.958943+00:00'}
Renderizando checkout para compra rápida
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:31:20.958943+00:00'}
Renderizando checkout para compra rápida
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:31:20.958943+00:00'}
Renderizando checkout para compra rápida
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:31:20.958943+00:00'}
Renderizando checkout para compra rápida
Error en guest_checkout: Invalid filter: 'intcomma'
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:54:11.591650+00:00'}
Renderizando checkout para compra rápida
Error en guest_checkout: Invalid filter: 'intcomma'
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:54:14.359279+00:00'}
Renderizando checkout para compra rápida
Error en guest_checkout: Invalid filter: 'intcomma'
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 06:55:41.225624+00:00'}
Renderizando checkout para compra rápida
Error en process_guest_order: Order() got unexpected keyword arguments: 'first_name', 'last_name', 'email', 'phone'
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 07:01:56.288473+00:00'}
Renderizando checkout para compra rápida
Error en guest_checkout: Invalid block tag on line 154: 'endblock'. Did you forget to register or load this tag?
Procesando checkout de compra rápida: {'product_id': 14, 'product_name': 'Monitor LG UltraGear 27"', 'product_price': 319992.0, 'quantity': 1, 'total': 319992.0, 'timestamp': '2024-11-26 07:18:47.712623+00:00'}
Renderizando checkout para compra rápida
Error en guest_checkout: Invalid block tag on line 154: 'endblock'. Did you forget to register or load this tag?
Iniciando proceso de guest checkout
Error en guest_checkout: Invalid block tag on line 154: 'endblock'. Did you forget to register or load this tag?
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 470, in guest_checkout
    return render(request, 'stock_smart/guest_checkout.html', context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 573, in invalid_block_tag
    raise self.error(
    ...<3 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 154: 'endblock'. Did you forget to register or load this tag?

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: no such table: stock_smart_payment
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: stock_smart_payment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 396, in guest_checkout
    payment = Payment.objects.create(
        variant='flow',
    ...<12 lines>...
        }
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 679, in create
    obj.save(force_insert=True, using=self.db)
    ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\payments\models.py", line 127, in save
    if not self.__class__._default_manager.filter(token=token).exists():
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 1288, in exists
    return self.query.has_results(using=self.db)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\query.py", line 660, in has_results
    return compiler.has_results()
           ~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1542, in has_results
    return bool(self.execute_sql(SINGLE))
                ~~~~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: stock_smart_payment

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: no such table: stock_smart_payment
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: stock_smart_payment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 396, in guest_checkout
    payment = Payment.objects.create(
        variant='flow',
    ...<12 lines>...
        }
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 679, in create
    obj.save(force_insert=True, using=self.db)
    ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\payments\models.py", line 127, in save
    if not self.__class__._default_manager.filter(token=token).exists():
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 1288, in exists
    return self.query.has_results(using=self.db)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\query.py", line 660, in has_results
    return compiler.has_results()
           ~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1542, in has_results
    return bool(self.execute_sql(SINGLE))
                ~~~~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: stock_smart_payment

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: Invalid filter: 'replace'
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 470, in guest_checkout
    return render(request, 'stock_smart/guest_checkout.html', context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 701, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 611, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'replace'

Iniciando proceso de guest checkout
Error en guest_checkout: Invalid filter: 'replace'
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 470, in guest_checkout
    return render(request, 'stock_smart/guest_checkout.html', context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 701, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\template\base.py", line 611, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'replace'

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: no such table: stock_smart_payment
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: stock_smart_payment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 396, in guest_checkout
    payment = Payment.objects.create(
        variant='flow',
    ...<12 lines>...
        }
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 679, in create
    obj.save(force_insert=True, using=self.db)
    ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\payments\models.py", line 127, in save
    if not self.__class__._default_manager.filter(token=token).exists():
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 1288, in exists
    return self.query.has_results(using=self.db)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\query.py", line 660, in has_results
    return compiler.has_results()
           ~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1542, in has_results
    return bool(self.execute_sql(SINGLE))
                ~~~~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: stock_smart_payment

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: no such table: stock_smart_payment
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: stock_smart_payment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 396, in guest_checkout
    payment = Payment.objects.create(
        variant='flow',
    ...<5 lines>...
        billing_email=form.cleaned_data['email']
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 679, in create
    obj.save(force_insert=True, using=self.db)
    ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\payments\models.py", line 127, in save
    if not self.__class__._default_manager.filter(token=token).exists():
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 1288, in exists
    return self.query.has_results(using=self.db)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\query.py", line 660, in has_results
    return compiler.has_results()
           ~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1542, in has_results
    return bool(self.execute_sql(SINGLE))
                ~~~~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: stock_smart_payment

Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en guest_checkout: no such table: stock_smart_payment
Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: stock_smart_payment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\stock_smart\views.py", line 396, in guest_checkout
    payment = Payment.objects.create(
        variant='flow',
    ...<5 lines>...
        billing_email=form.cleaned_data['email']
    )
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 679, in create
    obj.save(force_insert=True, using=self.db)
    ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\payments\models.py", line 127, in save
    if not self.__class__._default_manager.filter(token=token).exists():
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\query.py", line 1288, in exists
    return self.query.has_results(using=self.db)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\query.py", line 660, in has_results
    return compiler.has_results()
           ~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1542, in has_results
    return bool(self.execute_sql(SINGLE))
                ~~~~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Hernan-PC\Documents\proyecto titulo\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: stock_smart_payment

Iniciando proceso de guest checkout
Error en process_flow_payment: Order() got unexpected keyword arguments: 'first_name', 'last_name', 'email', 'phone', 'flow_token'
Error en process_flow_payment: Order() got unexpected keyword arguments: 'first_name', 'last_name', 'email', 'phone', 'flow_token'
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Error en process_flow_payment: Order() got unexpected keyword arguments: 'first_name', 'last_name', 'email', 'phone', 'flow_token'
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Redirigiendo a Flow: https://sandbox.flow.cl/app/web/pay.php?token=A68F8A0AFC7DB760CFFEEA05B84C90BF6930C7BH
Iniciando proceso de guest checkout
Redirigiendo a Flow: https://sandbox.flow.cl/app/web/pay.php?token=4030FB8628D76706B95C89F0280ABC95AE9BD67V
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
<<<<<<< Updated upstream
Iniciando proceso de pago con Flow
Error en process_flow_payment: No hay datos de checkout disponibles
Iniciando proceso de guest checkout
=======
Iniciando proceso de guest checkout
Iniciando proceso de guest checkout
Iniciando proceso de pago con Flow
Error en process_flow_payment: No hay datos de checkout disponibles
Iniciando proceso de pago con Flow
Error en process_flow_payment: No hay datos de checkout disponibles
Iniciando proceso de pago con Flow
Error en process_flow_payment: No hay datos de checkout disponibles
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['BQ1yGgLiuEcKlZENZrYY9x1fCsMgv4qoqS4EL3ZYntjGtcf0tg904Q5Nxhuwucrq'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['Pasaje de la merceria 1872']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Error al crear la orden: module 'datetime' has no attribute 'now'
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['myvIW7HDTmxEzInQiIUCIaaoyHmkXInrbAyO1UVjMbEAHVY3Mx5EDteWtw4AWQot'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Error al crear la orden: module 'datetime' has no attribute 'now'
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['BYWIUlT3HkpkYobA7wkTK9e8tB2Ut9xrq0ZOZ87JA9wg6BMNBlvVFsiGoqKashyt'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 1
ID de orden guardado en sesión: 1
Iniciando process_payment
ID de orden en sesión: 1
Orden encontrada: 271124001
No se pudo generar la URL de pago
Error en process_payment: Reverse for 'checkout_options' with arguments '(1,)' not found. 1 pattern(s) tried: ['checkout/options/\\Z']
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['QKK3RoUz2ntjZE22oea7tIpykTNpNrCyFMN9Wb8fVcAf7RDfS3l9o1t6fIvFMzDA'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 2
ID de orden guardado en sesión: 2
Iniciando process_payment
ID de orden en sesión: 2
Orden encontrada: 271124002
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['IidFBUIC5PsNWqHe882nYWJocTJU8w5nxkgLGHWiYEzJ4DirCXdpTfNW7Ira7E6p'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 3
ID de orden guardado en sesión: 3
Iniciando process_payment
ID de orden en sesión: 3
Orden encontrada: 271124003
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['bayAyARY0BncSlTjj4tcaw48nnsjwr6F0cBGDn5ETqu80yuwNTEe5P8Gicazvz7H'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 4
ID de orden guardado en sesión: 4
Iniciando process_payment
ID de orden en sesión: 4
Orden encontrada: 271124004
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['gN9ZY56LrgmCkKYAbLkyWBwbsMZEMDR05Pc53Skrk5tysXzNFAvARUAJnBHULLS2'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 5
ID de orden guardado en sesión: 5
Iniciando process_payment
ID de orden en sesión: 5
Orden encontrada: 271124005
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['uuQ9vjJWfcJSJOeQNlB0d6OqwJw2teChjwTfA6XC81QOR1P3haM28pSYryeismDj'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 6
ID de orden guardado en sesión: 6
Iniciando process_payment
ID de orden en sesión: 6
Orden encontrada: 271124006
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['z3fe7KMZyZhDBRzPPHRIHkdD3QlZHWHDo5ikcx0FrOozJ4a2jw2KCDhbYF3fG4IF'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 7
ID de orden guardado en sesión: 7
Iniciando process_payment
ID de orden en sesión: 7
Orden encontrada: 271124007
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['qvlp5pMoWWR63D74tXFXeE6IASqXEziPfxovac04PLY2bQIhXMQZ9XagvH8dDHjR'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 8
ID de orden guardado en sesión: 8
Iniciando process_payment
ID de orden en sesión: 8
Orden encontrada: 271124008
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['Fx3J3PWBQMLXi42giTjwlYawt5XHz3Dtuz6P8CahJBSTqhDtMIuyghe4oUFXybEv'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 9
ID de orden guardado en sesión: 9
Iniciando process_payment
ID de orden en sesión: 9
Orden encontrada: 271124009
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['8uDlrCI7QoViJOROVoxCrKbFuFNyejl6XwGrwpWNJd2eR1s1pdIEm3fdpuvOdrm8'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 10
ID de orden guardado en sesión: 10
Iniciando process_payment
ID de orden en sesión: 10
Orden encontrada: 271124010
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['BAVOB0sQ31Aa1LzagTDBVjkB10mVZ9i4qCYUGNGwWQH69YanKIODQCo9WP4bYhj6'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 11
ID de orden guardado en sesión: 11
Iniciando process_payment
ID de orden en sesión: 11
Orden encontrada: 271124011
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['UnbPrZ7TiXMXimUsCLQ0ylIH7ceWah3XJpeVwMlzbMTTqzvF6A12tEMf21Wc9p4Z'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 12
ID de orden guardado en sesión: 12
Iniciando process_payment
ID de orden en sesión: 12
Orden encontrada: 271124012
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['m5mjBqJMsC6SDeKo4A1ENRTI67g99eP1b7ppGdXslrdOLrlBypcGIaXg1WYp8mQ3'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 13
ID de orden guardado en sesión: 13
Iniciando process_payment
ID de orden en sesión: 13
Orden encontrada: 271124013
No se pudo generar la URL de pago
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Iniciando guest_checkout
Product ID from session: 1
Producto encontrado: xbox serie x
Procesando POST request
POST data: <QueryDict: {'csrfmiddlewaretoken': ['YUI8eEZYEEjZlfskSV6n8GSypdYFMjYINWLejrdExtqVts3xmKhp3ZW6k2GVLrZK'], 'nombre': ['Maria'], 'apellido': ['Mesa'], 'email': ['maria.mesa.neira@gmail.com'], 'telefono': ['934504484'], 'shipping': ['pickup'], 'direccion': ['']}>
Precios calculados: base=499990.00, final=349993.0000, total=349993.0000
Orden creada exitosamente: 14
ID de orden guardado en sesión: 14
Iniciando process_payment
ID de orden en sesión: 14
Orden encontrada: 271124014
URL de pago generada: https://sandbox.flow.cl/app/web/pay.php?token=3EFF8FFB6C097A424F1707042AFF15586C0EC95Z
Iniciando guest_checkout
Error en guest_checkout: Invalid block tag on line 187: 'endblock'. Did you forget to register or load this tag?
ID de producto en sesión: None
ID de producto en sesión: None
ID de producto en sesión: None
ID de producto en sesión: None
ID de producto en sesión: None
Product ID obtenido de la sesión: None
No se encontró product_id en POST ni en sesión
ID de producto en sesión: None
>>>>>>> Stashed changes
ID de producto en sesión: None
ID de producto en sesión: None
ID de producto en sesión: None
Producto ID guardado en sesión: 1
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Resumen de precios para producto xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Final: $349993.0000
            IVA: $55881.2352941176470588235294
        
Producto ID guardado en sesión: 1
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Resumen de precios para producto xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Final: $349993.0000
            IVA: $55881.2352941176470588235294
        
Producto ID guardado en sesión: 1
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Error en guest_checkout: Invalid filter: 'intcomma'
Producto ID guardado en sesión: 1
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
Recuperando producto ID de la sesión: 1
Producto encontrado: xbox serie x

            Precios calculados para xbox serie x:
            Base: $499990.00
            Descuento: 30.00%
            Monto descuento: $149997.0000
            Final: $349993.0000
            Sin IVA: $294111.7647058823529411764706
            IVA: $55881.2352941176470588235294
        
